packages/key-management/src/types.ts

```diff
@@ -38,10 +38,26 @@ export enum KeyRole {
   DRep = 3
 }
 
+export enum MultiSigKeyRole {
+  Payment = 0,
+  Stake = 2
+}
+
+export enum KeyPurpose {
+  INDIVIDUAL = 1852,
+  MULTI_SIG = 1854
+}
```

Comment:
This is a nitpick, but following the wording of CIP-1854

> To associate multi-signature keys to a wallet, we reserve however purpose=1854' to distinguish multisig wallets from standard wallets.

Maybe we could rename these:

```
export enum KeyPurpose {
  STANDARD = 1852,
  MULTI_SIG = 1854
}
```

Reply:
renamed on fac71bd